commands/dev-deploy.js

```diff
@@ -85,6 +84,10 @@ async function devDeploy(options) {
     // Deploy with init function and args
     const actions = [transactions.deployContract(readFileSync(options.wasmFile))];
 
+    if (options.initArgs && !options.initFunction) {
+        options.initFunction = 'new';
```

Comment:
Why we use this over `default: 'new'` ?

Reply:
@ailisp The idea is to be able not to set `initFunction` in CLI when it has the default name (new).
But when we are setting the default value of `initFunction` to new, args become mandatory even when you are not initializing the contract and it breaks the logic.